packages/scss/src/components/_card.scss

```diff
@@ -27,6 +27,6 @@ $cardPadding: 1.5rem;
 
 .card {
 	&.mod-grey {
-		background: $greyBackground;
+		background: #F5F5F5;
```

Comment:
For the moment I'm not sure if this color (which is used as body background color & in card component) should be defined on theming > commons or separately. 
Any advice?